appwrite/website
# Add git cli tips blog post and assets

## What does this PR do?

- Adds a new blog post: "15 Git command line tips every developer should know", along with its asset

## Test Plan

- Run the app and navigate to `/blog/post/15-git-cli-tips`
- Confirm that blog post and assets are displayed correctly
